### PR TITLE
Output rendition with dimensions == source

### PR DESCRIPTION
**Pull request by thomshutt**
This resolves https://github.com/livepeer/catalyst-api/issues/124

### CODE REVIEW

File: clients/broadcaster.go [hunk 1]
```diff
@@ -49,16 +49,16 @@ type StreamAllocResponse struct {
 
 type EncodedProfile struct {
 	Name         string `json:"name,omitempty"`
-	Width        int32  `json:"width,omitempty"`
-	Height       int32  `json:"height,omitempty"`
-	Bitrate      int32  `json:"bitrate,omitempty"`
-	FPS          uint   `json:"fps,omitempty"`
-	FPSDen       uint   `json:"fpsDen,omitempty"`
+	Width        int64  `json:"width,omitempty"`
```
Comment (thomshutt) [inline clients/broadcaster.go:52]: We were using different types in different structs, so standardised on int64 / float64

File: handlers/misttriggers/triggers_test.go [hunk 2]
```diff
@@ -89,7 +89,7 @@ func TestRecordingCompleted(t *testing.T) {
 		require.NotNil(t, message.Success)
 		require.True(t, *message.Success)
 		require.GreaterOrEqual(t, message.Timestamp, testStartTime)
-		require.Less(t, message.Timestamp, testStartTime+2)
+		require.Less(t, message.Timestamp, testStartTime+100)
```
Comment (thomshutt) [inline handlers/misttriggers/triggers_test.go:92]: Nothing changed here, but the timing was a bit too tight on CI and so I was seeing some failures

File: clients/broadcaster.go (hunk 1 above)
Comment (AlexKordic) [reply clients/broadcaster.go:52]: Go is much better than c++ when handling `int` and `uint` arithmetic !

File: transcode/transcode_test.go [hunk 3]
```diff
@@ -76,11 +76,11 @@ func TestItCanTranscode(t *testing.T) {
 		tr: clients.TranscodeResult{
 			Renditions: []*clients.RenditionSegment{
 				{
-					Name:      "lowlowlow",
+					Name:      "360p",
 					MediaData: []byte("pretend media data"),
 				},
 				{
-					Name:      "super-high-def",
+					Name:      "720p",
```
Comment (AlexKordic) [inline transcode/transcode_test.go:83]: Just a note for real broadcaster: name must start with letter otherwise transcode fails

**Review by AlexKordic — APPROVED** [review]
Well Done!

File: transcode/transcode_test.go (hunk 3 above)
Comment (thomshutt) [reply transcode/transcode_test.go:83]: I didn't realise, good to know thanks!

**Pull request merged**